### PR TITLE
factors out the updating doc into separate pages

### LINKED ISSUE #48: Refactor "Updating Bottlerocket" into two documents
In looking at the directory structure document and how the content currently in `/en/os/1.13.x/concepts/updating-bottlerocket/`, the current content doesn't quite meet the criteria of being 'high-level' since it has specific source and commands.

1. Move heading `#in-place-updates`, `#node-replacement`, and `#locking-to-a-specific-release` to separate files under the `/en/os/1.13.x/updating` directory.
2. Take the introduction text and list (starting with "There are many...") and turn that into a "Conclusion" section after the "ways to update section"
3. Remove the "Introduction" heading.

### PR DESCRIPTION
**Pull request by mchaker**
**Issue number:**

Closes #48 

**Description of changes:**

Refactors the Updating Bottlerocket concept document into three separate instructional documents:

- In-Place Updates
- Locking to a Specific Release
- Node Replacement

and cleans up the remaining content in the Updating Bottlerocket concept document.

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

**Review by stockholmux — CHANGES_REQUESTED** [review]
A few small changes - write a couple of intro sentences, move one bundle up a level, and explore a weird rendering issue.

File: content/en/os/1.13.x/update/methods/locking-to-a-specific-release/index.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
++++
+title = "Locking to a Specific Release"
```
[the remark this answers is not in the document]
Comment (mchaker) [reply content/en/os/1.13.x/update/methods/locking-to-a-specific-release/index.markdown:2]: I think you're right! I'll make that change.

File: content/en/os/1.13.x/concepts/updating-bottlerocket/index.markdown [hunk 2]
```diff
@@ -29,256 +18,24 @@ _In-place updates_ are updates that are applied to the same node (no infrastruct
...
-
-##### 1. Create the SSM Command Document for Updating Nodes
+[`apiclient` commands](../../update/methods/in-place/#apiclient-commands) | [EKS Console](../../update/methods/node-replacement/#eks-console)
+[ECS updater](../../update/methods/in-place/#ecs) | [`eksctl`](../../update/methods/node-replacement/#eksctl)
+[Brupop](../../update/methods/in-place/#brupop) |
+[SSM command documents](../../update/methods/in-place/#ssm) |
 
-To create an SSM Command Document, follow the steps in the [AWS Systems Manager User Guide: "Create an SSM document (console)"](https://docs.aws.amazon.com/systems-manager/latest/userguide/create-ssm-console.html).
-Remember to select "YAML" in the "Content" box, since the [SSM Command Document below](#ssm-command-document-check-for-and-apply-updates-to-a-bottlerocket-node) is formatted in YAML.
+## Conclusion
```
[the remark this answers is not in the document]
Comment (mchaker) [reply content/en/os/1.13.x/concepts/updating-bottlerocket/index.markdown:32]: Hmm... took a look at it, nothing seems to stand out compared to the other markdown files. Permissions are the same and filetype via `file` is the same.

[the remark this answers is not in the document]
Comment (stockholmux) [reply content/en/os/1.13.x/concepts/updating-bottlerocket/index.markdown:32]: Actually, it looks like it's rendering correctly now 🤷‍♂️

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**